src/components/GraphSection/Components/GraphError.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+
+const GraphError = () => {
+    console.log("")
+    return(
+        <div className="loading">Error...</div>
+)}
+
+export default React.memo(GraphError)
```

Comment:
We better avoid unnecessary use of memoization since it also adds performance overhead 
```suggestion
export default React.memo(GraphError)
```